docs/troubleshooting/management/ts-management.asciidoc

```diff
@@ -0,0 +1,33 @@
+[[ts-management]]
+== Management tools
+
+This topic covers common troubleshooting issues when using the {elastic-sec} <<sec-manage-intro, management tools>>.
+
+[discrete]
+[[ts-endpoints]]
+=== Endpoints
+
+[discrete]
+[[ts-transform-failed]]
+.Required transform failed
+[%collapsible]
+====
+If you encounter a `“Required transform failed”` notice on the Endpoints page, you can usually resolve the issue by restarting the transform. See {ref}/transforms.html[Transforming data] for more information on transforms.
+
+[role="screenshot"]
+image::../images/endpoints-transform-failed.png[Endpoints page with Required transform failed notice]
+
+To restart a transform that’s not running:
+
+. Go to *Stack Management -> Data -> Transforms*.
+. Search for `endpoint.metadata_current-default` in the search box to find the transform for {endpoint-sec}.
+. Click the *Actions* icon (*...*) and do one of the following, depending on the value in the *Status* column:
+* `stopped`: Select *Start* to restart the transform. 
+* `failed`: Select *Stop* to first stop the transform, and then select *Start* to restart it.
++
+[role="screenshot"]
+image::../images/transforms-start.png[Transforms page with Start option selected]
+
+. A confirmation message is displayed. Click *Start* to confirm.
```

Comment:
```suggestion
. Click *Start* in the confirmation message that displays to restart the transform.
```

Reply:
I only suggested this because all of the other steps, minus the last one, start with verbs. Also, the revision I suggested seems a tad long, so feel free to ignore it if it reduces the step's readability.